content/updates/2018-02.md

```diff
@@ -0,0 +1,23 @@
+---
+title: 2018.2.0
+date: 2018-03-07
```

Comment:
Date should be updated to deploy date, whenever that may be

Reply:
Fixed in 0680cac7447d72089d2d811aa2d990c86a4d6748

Reply:
(We're releasing today)